install.sh

```diff
@@ -66,26 +73,33 @@ case "${1}" in
         # install
         "-i" | "--install" )
         echo "Welcome to the install-script for adaway-linux."
+        # check root
+        if [ "${UID}" != "0" ] ; then
+          echo "[!] For this action the script must be run as root." 1>&2
+          exit 1
+        fi
         echo "[!] Please run this only ONCE! Cancel, if you already modified /etc/hosts by adaway-linux.sh."
         read -r -p "[?] Proceed? [Y/n] " REPLY
         case "${REPLY}" in
             [Yy][Ee][Ss] | [Yy] | "" ) # YES, Y, NULL
-                # check root
-                if [ "${UID}" != "0" ] ; then
-                  echo "[!] For this action the script must be run as root" 1>&2
-                  exit 1
-                fi
 
                 # check if script wasn't started with the -f option
-                if [ "$2" != "-f" ] && [ "$ARG1" != "--force" ] ; then
+                if [ "${2}" != "-f" ] && [ "${2}" != "--force" ] ; then
                     # backup /etc/hosts
                     echo "[i] First I will backup the original /etc/hosts to ${HOSTS_ORIG}."
+                    # checks if /etc/.hosts.original already exist
+                    if [ -e ${HOSTS_ORIG} ] ; then
+                      echo "[!] Backup of /etc/hosts already exist. To uninstall run: »${0} -u«"
+                      exit 1
+                    fi
                     cp /etc/hosts "${HOSTS_ORIG}"
                     # check if backup was succesfully
                     if [ ! -e "${HOSTS_ORIG}" ] ; then
                         echo "[!] Backup of /etc/hosts failed. Please backup this file manually and bypass this check by using the -f parameter."
                         exit 1
                     fi
+                else
+                  rm -f ${HOSTS_ORIG}
```

Comment:
Controversial whether the `/etc/.hosts.original` should be deleted when the user uses the --force option or not.

Reply:
I don't have a opinion on this 🤷‍

Reply:
Well then I just leave it. It'll remove the broken `.hosts.original` if it existed before.